bitnami/kafka/values.yaml

```
@@ -1366,6 +1366,48 @@ provisioning:
   ## @param provisioning.enabled Enable kafka provisioning Job
   ##
   enabled: false
+  ## Auth Configuration for kafka provisioning Job
+  ##
+  auth:
+    ## TLS configuration for kafka provisioning Job
+    ##
+    tls:
+      ## @param provisioning.auth.tls.type Format to use for TLS certificates. Allowed types: `jks` and `pem`.
+      ## Note: ignored if auth.tls.clientProtocol different from one of these values: "tls" "mtls" "sasl_tls".
+      ##
+      type: jks
+      ## @param provisioning.auth.tls.certificatesSecret Existing secret containing the TLS certificates for the Kafka provisioning Job.
+      ## When using 'jks' format for certificates, the secret should contain a truststore and a keystore.
+      ## When using 'pem' format for certificates, the secret should contain a public CA certificate, a public certificate and one private key.
+      ##
+      certificatesSecret: ""
+      ## @param provisioning.auth.tls.cert The secret key from the certificatesSecret if 'cert' key different from the default (tls.crt)
+      ##
+      cert: tls.crt
+      ## @param provisioning.auth.tls.key The secret key from the certificatesSecret if 'key' key different from the default (tls.key)
+      ##
+      key: tls.key
+      ## @param provisioning.auth.tls.caCert The secret key from the certificatesSecret if 'caCert' key different from the default (ca.crt)
+      ##
+      caCert: ca.crt
+      ## @param provisioning.auth.tls.keystore The secret key from the certificatesSecret if 'keystore' key different from the default (keystore.jks)
+      ##
+      keystore: keystore.jks
+      ## @param provisioning.auth.tls.truststore The secret key from the certificatesSecret if 'truststore' key different from the default (truststore.jks)
+      ##
+      truststore: truststore.jks
+      ## @param provisioning.auth.tls.existingSecret Name of the secret containing passwords to access the JKS files or PEM key when they are password-protected.
+      ##
+      existingSecret: ""
```

To be aligned with other charts, I would appreciate it if you could rename it as passwordsSecrets
```suggestion
      ## @param provisioning.auth.tls.passwordsSecret Name of the secret containing passwords to access the JKS files or PEM key when they are password-protected.
      ## It should contain two keys called "keystore-password" and "truststore-password", or "key-password" if using a password-protected PEM key.
      ##
      passwordsSecret: ""
```